packages/web5-user-agent/src/utils.ts

```diff
@@ -1,10 +1,40 @@
 import type { Readable } from 'readable-stream';
 import { ReadableWebToNodeStream } from 'readable-web-to-node-stream';
+import { Encoder} from '@tbd54566975/dwn-sdk-js';
 
 export function blobToIsomorphicNodeReadable(blob: Blob): Readable {
   return webReadableToIsomorphicNodeReadable(blob.stream());
 }
 
 export function webReadableToIsomorphicNodeReadable(webReadable: ReadableStream) {
   return new ReadableWebToNodeStream(webReadable);
+}
+
+export function dataToBlob(data, dataFormat){
```

Comment:
Per the other comment about moving the `Blob` conversion to `VcApi`, since its already in `packages/web5/src/utils`, you can import from there in the `web5` package and remove this from `web5-user-agent`.